### PR TITLE
chore: update snowcrash version (fix crash - undislosed item list)

### PR DESCRIPTION
**Pull request by klokane**

### CODE REVIEW

File: test/fixtures/syntax/undisclosed-listitem.json [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+                  "element": "dataStructure",
+                  "content": [
+                    {
+                      "element": "object",
+                      "content": [
+                        {
+                          "element": "member",
+                          "content": {
+                            "key": {
+                              "element": "string"
```
Comment (kylef) [inline test/fixtures/syntax/undisclosed-listitem.json]: A key without content. Should we perhaps issue a warning to the user that they didn't supply a key for an MSON property?

Comment (klokane) [inline test/fixtures/syntax/undisclosed-listitem.json]: it is original behavior (before sundown changes), but I can introduce fix for it if you like

Comment (kylef) [inline test/fixtures/syntax/undisclosed-listitem.json]: No need, just a though.

Comment (pksunkara) [inline test/fixtures/syntax/undisclosed-listitem.json]: I am not sure, but IIRC we had a discussion on that and decided to leave it for now.

**Pull request merged**